### PR TITLE
Add reference to AutoWarp

### PR DESCRIPTION
**Pull request by christophetd**

### CODE REVIEW

**Comment by yuvalavra** [comment]
This seems Critical to me

**Comment by christophetd** [comment]
Didn't see there was other "Critical" ones, I thought "High" was the highest currently

File: README.md [hunk 1]
```diff
@@ -552,3 +552,13 @@ AWS access key. The initial signing algorithm, SigV1, was vulnerable to collisio
 - Customer action: Use KMS-CMK for bucket encryption, Use Least privilege on resources policies (Buckets)
 - References:
   - https://aws.amazon.com/security/security-bulletins/AWS-2021-007/
+
+### Azure: AutoWarp
+
+- Summary: An exposed endpoint in the Azure Automation Service allowed to steal Azure API credentials from other customers
+- Platform: Azure
+- Severity: High
```
Comment (christophetd) [inline README.md:560]:
```suggestion
- Severity: Critical
```

**Comment by 0xdabbad00** [comment]
Thank you!

**Pull request merged**